src/pybind/mgr/plugins/more_itertools.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) 2012 Erik Rose
```

Comment:
Ais I said, the copyright is wrong. it originates from https://docs.python.org/3/library/itertools.html#itertools-recipes instead.

Reply:
Removed the copyright